README.md

```diff
@@ -1,22 +1,76 @@
 # LDOLayoutTemplates
 
-Design different states of a screen in IB and easily transition between them
+Visually design multiple layouts of a view in IB and easily transition between them
 
 [![Version](https://img.shields.io/cocoapods/v/LDOLayoutTemplates.svg?style=flat)](http://cocoapods.org/pods/LDOLayoutTemplates)
 [![License](https://img.shields.io/cocoapods/l/LDOLayoutTemplates.svg?style=flat)](http://cocoapods.org/pods/LDOLayoutTemplates)
 [![Platform](https://img.shields.io/cocoapods/p/LDOLayoutTemplates.svg?style=flat)](http://cocoapods.org/pods/LDOLayoutTemplates)
 
+### Get this
+
+![Demo](Screenshots/ComplexExample.gif)
+
+### With this
+
+```Swift
+class ComplexViewController: UIViewController {
+    @IBOutlet weak var portraitLargeEventsLayout: LDOLayoutTemplate!
+    @IBOutlet weak var landscapeLargeEventsLayout: LDOLayoutTemplate!
+    private var defaultLayout: LDOLayoutTemplate!
+    private var largeBlueLayoutActive = false
```

Comment:
What do you think about moving the transition from the `IBAction` to a `didSet` clause here?

Reply:
Plus: the `largeListLayoutActive` bool and the applied layout are more in sync
Minus: the code is more spread out and fancier
Neutral: I dislike heavy `didSet`s and changing the layout is at least close to that category